README.md

```
@@ -44,7 +45,7 @@ $ PARALLEL_WORKERS=1 rails test:all
 次のコマンドでlintを実行します。
 
 ```
-$ ./bin/lint
+$ bin/lint
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)